SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/UNR-3761/SpatialTestNetReference/SpatialTestNetReference.cpp

```diff
@@ -60,8 +60,6 @@ void ASpatialTestNetReference::PrepareTest()
 {
 	Super::PrepareTest();
 
-	PreviousMaximumDistanceThreshold = GetDefault<USpatialGDKSettings>()->PositionUpdateThresholdMaxCentimeters;
```

Comment:
Where is this defined now? Don't we need an ini file for this test?

Reply:
You are right this is now in the ini file you will find it in the corresponding TestGym PR: Game/Config/MapSettingsOverrides/TestOverridesSpatialNetReferenceMap.ini

Reply:
Oh, seems bad to split these, can we include them alongside the map data in the GDK?

Reply:
Miron has a fix for generated test maps to create them in the:
`Samples\UnrealGDKTestGyms\Game\Intermediate\Config\MapSettingsOverrides` folder
But ones created manually will still reside: \Samples\UnrealGDKTestGyms\Game\Config\MapSettingsOverrides